### ISSUE TITLE
Path is not correct on mac OS

### ISSUE THREAD

**Issue by mymattcarroll** [vweevers/dotnet-bump#1]
Command:

```
dotnet-bump 1.0.0-beta.1 --dry-run

ENOENT: no such file or directory, open '/Users/me/project/Project\Project.csproj'
```

**Comment by vweevers** [comment]
Is there a solution file (`*.sln`) in the current working directory that perhaps contains a path with a backwards slash?

**Comment by mymattcarroll** [comment]
Yes there is, I have updated to a forward slash and am now getting this error:

```
dotnet-bump 1.0.0-beta.1 --dry-run
ENOTDIR: not a directory, stat '/Users/me/project/Project/Project.csproj/.git
```

I am running the command from `/Users/me/project`

**Comment by vweevers** [comment]
> Yes there is, I have updated to a forward slash

OK. We can add path normalization to `dotnet-bump` if you want.

> now getting this error: [..]

Can you run `dotnet-bump 1.0.0-beta.1 --dry-run --verbose`? Will give us a stack trace.

**Comment by mymattcarroll** [comment]
```sh
npx dotnet-bump 1.0.0-beta.1 --dry-run --verbose
npx: installed 110 in 9.178s
/Users/me/.npm/_npx/27403/lib/node_modules/dotnet-bump/cli.js:65
    throw err
    ^

[Error: ENOTDIR: not a directory, stat '/Users/me/project/Project/Project.csproj/.git'] {
  errno: -20,
  code: 'ENOTDIR',
  syscall: 'stat',
  path: '/Users/me/project/Project/Project.csproj/.git'
}
```

**Comment by vweevers** [comment]
Hm, that stack trace is too short. Would it be possible for you to provide a (stripped-down) repository, so I can reproduce?

**Comment by mymattcarroll** [comment]
Ill try and put something together this week.

**Comment by mymattcarroll** [comment]
```sh
cd /Users/me/sources
```

```sh
git clone git@github.com:mymattcarroll/dotnet-bump-test.git
```

```sh
cd dotnet-bump-test
```

```sh
npm install --global dotnet-bump
```

```sh
dotnet-bump 1.1.1 --verbose
```

### Output:

```
/Users/me/.npm-packages/lib/node_modules/dotnet-bump/cli.js:65
    throw err
    ^

[Error: ENOTDIR: not a directory, stat '/Users/me/sources/dotnet-bump-test/project.csproj/.git'] {
  errno: -20,
  code: 'ENOTDIR',
  syscall: 'stat',
  path: '/Users/me/sources/dotnet-bump-test/project.csproj/.git'
}
```

**Comment by vweevers** [comment]
Excellent, thanks! Will have a look later this week.

**Comment by vweevers** [comment]
Which node version do you have?

**Comment by mymattcarroll** [comment]
Node v14.15.1
NPM v6.14.11

**Comment by benbelow** [comment]
> ENOTDIR: not a directory, stat '/Users/me/project/Project/Project.csproj/.git

I'm getting essentially the same error when trying to run this on a Ubuntu host as part of a github action

**Comment by vweevers** [comment]
Fixed in [1.5.0](https://github.com/vweevers/dotnet-bump/releases/tag/v1.5.0)
